### PR TITLE
Update README to add campaigns editor instructions

### PR DESCRIPTION
**Pull request by domdomegg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,20 @@ Stores data about Commons campaigns such as Wiki Loves Monuments.
...
 Campaigns will be displayed by the [Commons Android app](https://github.com/commons-app/apps-android-commons/) using this information.
 
-# Adding a campaign
+## Adding a campaign in the campaign editor (recommended)
+
+If a campaign is not documented yet, or to modify the dates or URL, please:
+
+1. Go to the [campaign editor](https://commons-app.github.io/campaigns-editor/)
+2. Click `Import` > `Live version`, then click `Add` and enter the details for the campaign
+3. Scroll down to the result, click `Copy to clipboard` and then `Edit on Github`
```
Comment (nicolas-raoul) [inline README.md]: GitHub

File: README.md [hunk 2]
```diff
@@ -4,7 +4,20 @@ Stores data about Commons campaigns such as Wiki Loves Monuments.
...
+4. Delete the existing content, and paste in the updated campaigns file
+5. In the pull request description, provide a source for the information, for instance a link to an announcement or wiki discussion.
+6. We will review your contribution and merge it.
```
Comment (nicolas-raoul) [inline README.md]: Some of the sentences in this list end with a point, some do not. Maybe end all with a point? :-)

File: README.md (hunk 1 above)
Comment (domdomegg) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (domdomegg) [reply README.md]: Fixed

**Comment by domdomegg** [comment]
Thanks for the really quick review! Think it should be fixed now.

**Pull request merged**